larrecodnn/ImagePatternAlgs/Modules/CMakeLists.txt

```diff
@@ -20,6 +21,7 @@ art_make( MODULE_LIBRARIES
                        art::Utilities
                        canvas::canvas
                        messagefacility::MF_MessageLogger
+                       ${CLHEP}
```

Comment:
```suggestion
                       CLHEP::CLHEP
```

Reply:
thanks Kyle, I made this change in CMakeLists.txt.